test/e2e/egress_firewall.go

```diff
@@ -52,6 +54,7 @@ var _ = ginkgo.Describe("e2e egress firewall policy validation", func() {
 	)
 
 	f := wrappedTestFramework(svcname)
+	node2ndaryIPs := make(map[string]map[int]string)
```

Comment:
comment saying what the keys/values are here for this map?

Reply:
yep done.